group_vars/all

```diff
@@ -35,7 +35,6 @@ rvm1_rvm_version: '1.29.4'
 rvm1_rvm_check_for_updates: false
 
 #Postgresql
-postgresql_version: 9.6
```

Comment:
It will now just use [the latest version from the Ubuntu package index](https://packages.ubuntu.com/xenial/postgresql)... Is 9.6 needed specifically?

Reply:
Cool >= 9.6 should be fine 👍